fork.html

```diff
@@ -0,0 +1,170 @@
+            available at <a href="https://github.com/opentffoundation/opentf">github.com/opentffoundation/opentf</a>!</strong>
+          </p>
+          <h3 class="small-margin">What is a "preview?"</h3>
+          <p class="subtle-margin">
+            By "preview," we mean that the fork is now public, but there is more work to do before we can put official 
+            releases out there for you to use. <strong>The goal of this preview release is to start doing this work in 
+            the open.</strong>
+          </p>
+          <h3 class="small-margin">Working in the open</h3>
+          <p class="subtle-margin">
+            Our goal with OpenTF is to create a project that is truly open source, community-driven, and impartial. To
+            that end, going forward, we'll be developing OpenTF in the open. We had to do some work on the fork and the
+            OpenTF foundation in private to get everything ready for public consumption, but now that that's done,
+            with this preview release, you'll be able to see everything we're working on—and start to participate 
+            yourself!
+          </p>
+          <h3 class="small-margin" id="open-roadmap">An open roadmap: the path to stable OpenTF releases</h3>
+          <p class="subtle-margin">
+            So, how do we get from the "preview" to where we have OpenTF releases available for everyone? A key part of 
+            working in the open is making our roadmap open. So here's a quick snapshot of what we already got done, 
+            what's in progress now, and what's coming up in the future. 
+          </p>
+          <h4 class="small-margin">What we already got done</h4>
+          <ul class="tight-list">
+            <li><strong>✅ Publish the OpenTF manifesto</strong>. We published the OpenTF manifesto at <a href="https://opentf.org/">opentf.org</a>.</li>  
+            <li><strong>✅ Wait on HashiCorp's response</strong>. We reached out to HashiCorp publicly and privately and requested a response by August 25th.</li>  
+            <li><strong>✅ Start working on the OpenTF fork</strong>. With no response from HashiCorp, we created the OpenTF fork, and started working on it in private.</li>  
+            <li><strong>✅ Apply to join the Linux Foundation</strong>. We want OpenTF to be part of an impartial, community-driven foundation, so we submitted all the paperwork to join the Linux Foundation.</li>  
+            <li><strong>✅ Open up community discussions</strong>. We opened up <a href="https://github.com/orgs/opentffoundation/discussions">OpenTF Discussions on GitHub</a> to give the community a way to have discussions, provide feedback, ask questions, etc.</li>  
+            <li><strong>✅ Release the OpenTF fork preview</strong>. As per this announcement, we are making the OpenTF fork preview public at <a href="https://github.com/opentffoundation/opentf">github.com/opentffoundation/opentf</a>!</li>
```

Comment:
fork preview => repository

in this whole point.

Reply:
Done